packages/svelte/src/main/private.d.ts

```diff
@@ -1,4 +1,16 @@
+import type { Snippet } from './public';
+
 /**
  * Anything except a function
  */
 export type NotFunction<T> = T extends Function ? never : T;
+
+// Utility type for ensuring backwards compatibility on a type level: If there's a default slot, add 'children' to the props if it doesn't exist there already
+// if you're curious why this is here and not declared as an unexported type from `public.d.ts`, try putting it there
+// and see what happens in the output `index.d.ts` -- it breaks, presumably because of a TypeScript compiler bug.
```

Comment:
is this still true? i just tried it, and it seems to work fine. how would i see it failing?